csharp/Platform.Numbers/Bit[T].cs

```diff
@@ -9,50 +9,50 @@
 namespace Platform.Numbers
 {
     /// <summary>
-    /// <para>.</para>
-    /// <para>.</para>
+    /// <para>Represents a set of compiled bit operation delegates.</para>
+    /// <para>Представляет набор скомпилированных делегатов битовых операций.</para>
     /// </summary>
     public static class Bit<T>
     {
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>A read-only field that represents a delegate of the function of the bitwise inversion of a number.</para>
```

Comment:
`delegate of the function of the bitwise inversion` ⇾ `bitwise inversion function delegate`